src/clients/BundleDataClient/utils/DataworkerUtils.ts

```diff
@@ -160,8 +161,10 @@ export function _buildPoolRebalanceRoot(
           mainnetBundleEndBlock
         );
 
-        updateRunningBalance(runningBalances, repaymentChainId, l1TokenCounterpart, totalRefundAmount);
-        updateRunningBalance(realizedLpFees, repaymentChainId, l1TokenCounterpart, totalRealizedLpFee);
+        if (isDefined(l1TokenCounterpart)) {
```

Comment:
Leaving a comment here since this is just a temporary fix to let the code compile. I'm pretty sure this is a bad idea to do since if `l1TokenCounterpart` is undefined, then we probably do want to throw an error. (Maybe we should just assert that here).